common/ECSResourcesCheck/src/main/java/Writer.java

```diff
@@ -0,0 +1,68 @@
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Date;
+
+public class Writer {
+    public void write(JSONObject json, Date date, String cluster)
+            throws IOException {
+        String Head = "<style type=\"text/css\">\n" +
+                ".tg  {border-collapse:collapse;border-color:#ccc;border-spacing:0;}\n" +
+                ".tg td{background-color:#fff;border-color:#ccc;border-style:solid;border-width:0px;color:#333;\n" +
+                "  font-family:Arial, sans-serif;font-size:14px;overflow:hidden;padding:10px 3px;word-break:normal;}\n" +
+                ".tg th{background-color:#f0f0f0;border-color:#ccc;border-style:solid;border-width:0px;color:#333;\n" +
+                "  font-family:Arial, sans-serif;font-size:14px;font-weight:normal;overflow:hidden;padding:10px 3px;word-break:normal;}\n" +
+                ".tg .tg-73oq{border-color:#000000;text-align:left;vertical-align:top}\n" +
+                "</style>\n" +
+                "<table class=\"tg\">\n" +
+                "    <thead>\n" +
+                "    <tr>\n" +
+                "        <th class=\"tg-73oq\">Module<br>"+cluster+"<br>  "+date+"</th>\n" +
+                "        <th class=\"tg-73oq\">Revision</th>\n" +
+                "        <th class=\"tg-73oq\">Version</th>\n" +
+                "        <th class=\"tg-73oq\">Count</th>\n" +
+                "        <th class=\"tg-73oq\"><span style=\"font-weight:400;font-style:normal\">Hard LImit</span></th>\n" +
+                "        <th class=\"tg-73oq\"><span style=\"font-weight:400;font-style:normal\">Soft limit</span></th>\n" +
```

Comment:
May as well throw this style into a class and refer to it like you are doing elsewhere.